code/modules/projectiles/guns/mods/mods.dm

```diff
@@ -831,7 +831,7 @@
 	I.removal_time *= rand(10, 14)/10
 	I.removal_difficulty *= rand(5, 15)/10
 	I.gun_loc_tag = GUN_MUZZLE
-	I.prefix = "silenced"
+	I.prefix = "badly silenced"
```

Comment:
```suggestion
	I.prefix = "silenced"
```
its meant to trick people into use it

Reply:
Eh, at this point everyone knows that no (good) weapon comes pre-modded but sure